back/app/services/side_fx_idea_service.rb

```diff
@@ -161,7 +161,7 @@ def sanitize_location_point(change)
   end
 
   def log_activities_if_cosponsors_added(idea, user, old_cosponsor_ids)
-    added_ids = idea.reload.cosponsors.map(&:id) - old_cosponsor_ids
+    added_ids = idea.cosponsors.map(&:id) - old_cosponsor_ids
```

Comment:
What did the reload do exactly?

Reply:
I think it was just a precaution to have the lastest data, but it broke ActiveModel::Dirty methods for the check on changed assignee. I couldn't find an issue with not reloading so think this is fine.